examples/with-tailwind/packages/ui/package.json

```diff
@@ -9,12 +9,8 @@
   },
   "license": "MIT",
   "scripts": {
-    "build": "run-p \"build:*\"",
-    "dev": "run-p \"dev:*\"",
-    "build:styles": "tailwindcss -i ./src/styles.css -o ./dist/styles.css",
-    "dev:styles": "tailwindcss -i ./src/styles.css -o ./dist/styles.css --watch",
-    "build:source": "tsup src/index.tsx --format esm,cjs --dts --external react",
-    "dev:source": "tsup src/index.tsx --format esm,cjs --dts --external react --watch",
+    "build": "tsup src/index.tsx --format esm,cjs --dts --external react && tailwindcss -i ./src/styles.css -o ./dist/styles.css",
+    "dev": "tsup src/index.tsx --format esm,cjs --dts --external react --watch & tailwindcss -i ./src/styles.css -o ./dist/styles.css --watch",
```

Comment:
This is incorrect imho. The tailwind styles compilation should be moved into the next.js apps and compiled at the _end_ at the last mile in JIT. Should not need this interim step

Reply:
Instead, make the next.js app's tailwind config aware that `ui` package needs to be included in scanning `content` array